<ui version="4.0" >
 <class>CommandDialogClass</class>
 <widget class="QDialog" name="CommandDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CommandDialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>3</number>
   </property>
   <property name="topMargin" >
    <number>3</number>
   </property>
   <property name="rightMargin" >
    <number>3</number>
   </property>
   <property name="bottomMargin" >
    <number>3</number>
   </property>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lblInfo" >
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="wdtForm" />
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QWidget" native="1" name="wdtPages" />
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dbbButtons" >
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
